src/Profile.js

```diff
@@ -1,53 +1,42 @@
-import React, {useEffect, useState} from 'react';
-import {useNavigate} from "react-router-dom";
-import Corbado from "@corbado/webcomponent";
+import { useNavigate } from "react-router-dom"
+import { useCorbado } from "@corbado/react"
 
-const CORBADO_PROJECT_ID = process.env.REACT_APP_CORBADO_PROJECT_ID;
-
-function Profile() {
-    const navigate = useNavigate();
-    const [user, setUser] = useState(null);
-
-    useEffect(() => {
-    const session = new Corbado.Session(CORBADO_PROJECT_ID);
-        session.refresh(user => {
-            setUser(user);
-        });
-    }, []);
-
-    const session = new Corbado.Session(CORBADO_PROJECT_ID);
-    const handleLogout = () => {
-        session.logout()
-            .then(async () => {
-                navigate("/");
-            })
-            .catch(err => console.log(err))
-    }
+export default function Profile() {
+    const navigate = useNavigate()
+    const { shortSession, user, logout } = useCorbado()
 
     const redirectToHome = () => {
-        navigate("/");
+        navigate("/")
+    }
+    const handleLogout = () => {
+        logout()
+        redirectToHome()
     }
 
-    if (user) {
+    if (user && shortSession) {
```

Comment:
@incorbador do we need this shortSession here at all. Shouldn't it be fine to just check for the user object not being null?

Reply:
@vincentdelitz  TODO copy from the React hooks in the wizard - the full example